autometa.py

```diff
@@ -137,7 +141,6 @@ def main(args):
         main(args)
     except KeyboardInterrupt:
         logger.info('User cancelled run. Exiting...')
-        sys.exit(1)
```

Comment:
Why is this line deleted?

Reply:
In short, It is unnecessary. At this point in the code, the error is handled and the next action will be to exit.